### PR TITLE
docs: fixed documentation typos for Hover directive

### PR DESCRIPTION
**Pull request by jreiersgord**
### Describe the PR

This pull request simply contains 2 grammar fixes for the Hover directive in the documentation.

### PR checklist

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Enhancement
- [ ] ARIA accessibility
- [x] Documentation update
- [ ] Other (please describe)

**Does this PR introduce a breaking change?** (check one)

- [x] No
- [ ] Yes (please describe)

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch, **not** the `master` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (i.e. `[...] (fixes #xxx[,#xxx])`, where "xxx" is the issue number)
- [x] It should address only one issue or feature. If adding multiple features or fixing a bug and adding a new feature, break them into separate PRs if at all possible.
- [x] The title should follow the [**Conventional Commits**](https://www.conventionalcommits.org/) naming convention (i.e. `fix(alert): not alerting during SSR render`, `docs(badge): update pill examples`, `chore(docs): fix typo in README`, etc). **This is very important, as the `CHANGELOG` is generated from these messages.**

**If new features/enhancement/fixes are added or changed:**

- [ ] Includes documentation updates (including updating the component's `package.json` for slot and event changes)
- [ ] Includes any needed TypeScript declaration file updates
- [ ] New/updated tests are included and passing (if required)
- [ ] Existing test suites are passing
- [ ] CodeCov for patch has met target
- [ ] The changes have not impacted the functionality of other components or directives
- [ ] ARIA Accessibility has been taken into consideration (Does it affect screen reader users or keyboard only users? Clickable items should be in the tab index, etc.)

**If adding a new feature, or changing the functionality of an existing feature, the PR's
description above includes:**

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

File: src/directives/hover/README.md [hunk 1]
```diff
@@ -84,4 +84,4 @@ In the following, we are swapping icons and tet color depending on the hover sta
 ## Accessibility concerns
 
 Hover state should not be used to convey special meaning, as screen reader users and keyboard only
-users typically ac not typically trigger hover state on elements.
+users typically do not trigger hover state on elements.
```
Comment (tmorehouse) [inline src/directives/hover/README.md:87]: Actually this was supposed to be `users typically can not trigger`... could you update it?

File: src/directives/hover/README.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ The directive has no modifiers.
 
 ## Live example
 
-In the following, we are swapping icons and tet color depending on the hover state of the element:
```
Comment (jackmu95) [inline src/directives/hover/README.md:53]: @tmorehouse This is the first time I see this form of typo for `the`.
Good to know - so I can add it to the list of things I have to look for 😅

Comment (tmorehouse) [reply src/directives/hover/README.md:53]: Oops that should be `text color` and not `the color` (bad `x` key on the keyboard)

Comment (tmorehouse) [reply src/directives/hover/README.md:53]: I have fixed it.

**Comment by tmorehouse** [comment]
Thanks @jreiersgord !

**Comment by jreiersgord** [comment]
Ah, I was wrong too. ;)

Thank you!

**Pull request merged**